src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        System.out.println(firstBall.toString());
```

Comment:
Calling `toString()` is redundant because `println` internally calls the `toString()` method of the passed object. You can simply pass the `firstBall` object to `println`.